db/model/user.js

```diff
@@ -52,6 +52,10 @@ module.exports = function user(seq, dataTypes) {
       type: dataTypes.BOOLEAN,
       defaultValue: false,
     },
+    fullName: {
+      type: dataTypes.STRING,
```

Comment:
let's use one of the db constants for string length -- a name should not be unlimited length!